.vitepress/theme/style.css

```
@@ -0,0 +1,298 @@
+  --vp-button-brand-hover-text: var(--vp-c-white);
+  --vp-button-brand-hover-bg: var(--kyma-color-primary-dark);
+  --vp-button-brand-active-border: transparent;
+  --vp-button-brand-active-text: var(--vp-c-white);
+  --vp-button-brand-active-bg: var(--vp-c-brand-1);
+}
+
+/**
+ * Component: Home
+ * -------------------------------------------------------------------------- */
+
+:root {
+  --vp-home-hero-name-color: transparent;
+  --vp-home-hero-name-background: -webkit-linear-gradient(
+    120deg,
+    #014ea0 20%,
+    var(--kyma-color-primary) 60%
+  );
+
+  --vp-home-hero-image-background-image: linear-gradient(
+    -45deg,
+    #01366f 50%,
+    var(--kyma-color-primary) 50%
+  );
+  --vp-home-hero-image-filter: blur(44px);
+}
+
+@media (min-width: 640px) {
+  :root {
+    --vp-home-hero-image-filter: blur(56px);
+  }
+}
+
+@media (min-width: 960px) {
+  :root {
+    --vp-home-hero-image-filter: blur(68px);
+  }
+}
+
+span.home-kyma-heavy-bold {
+    font-weight: bold;
+    color: var(--kyma-color-primary);
+}
+
+span.home-kyma-phonetic {
+    font-style: italic;
+    color: #0195FF;
+}
+
+
+/**
+ * Component: Custom Block
+ * -------------------------------------------------------------------------- */
+
+:root {
+  --vp-custom-block-tip-border: transparent;
+  --vp-custom-block-tip-text: var(--vp-c-text-1);
+  --vp-custom-block-tip-bg: var(--vp-c-brand-soft);
+  --vp-custom-block-tip-code-bg: var(--vp-c-brand-soft);
+}
+
+/**
+ * Component: Algolia
+ * -------------------------------------------------------------------------- */
+
+.DocSearch {
+  --docsearch-primary-color: var(--vp-c-brand-1) !important;
+}
+
+/* Custom styles for features grid */
+@media (min-width: 768px) {
+  .VPHomeFeatures {
+    width: 100vw;
+    margin-left: calc(-50vw + 50%);
+  }
+
+  .VPHomeFeatures .items {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 2rem;
+    width: 100%;
+    max-width: none; 
+  }
+
+  .VPHomeFeatures .item {
+    width: 100% !important;
+  }
+
+  .VPHomeFeatures .box:hover {
+    transform: scale(1.1);
+  }
+
+  .VPHomeFeatures .box {
+  position: relative;
+  overflow: hidden;
+  border-radius: 12px;
+  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.05);
+  background-color: var(--vp-c-bg);
+  transition: transform 0.6s ease;
+}
+
+.VPHomeFeatures .box::before {
+  content: "";
+  position: absolute;
+  inset: 0;
+  background: radial-gradient(
+    var(--kyma-color-primary-dark) 0%,
+    var(--kyma-color-primary) 60%,
+    transparent 100%
+  );
+  opacity: 0;
+  transition: opacity 0.6s ease;
+  filter: blur(150px);
+  z-index: 0;
+}
+
+.VPHomeFeatures .box:hover::before {
+  opacity: 1;
+}
+
+.VPHomeFeatures .box > * {
+  position: relative;
+  z-index: 1;
+}
+
+}
+
+.VPFeature a {
+  color: var(--kyma-color-primary); /* Change to your preferred link color */
+  text-decoration: none; /* Removes underline */
+  text-decoration: underline; /* Adds underline on hover */
+}
```

Also in my opinion most of this comments are unnecesary, and they is no added value with them.
```suggestion
.VPFeature a {
  color: var(--kyma-color-primary); 
  text-decoration: none; 
  text-decoration: underline; 
}
```